.github/workflows/test_unreleased.yml

```diff
@@ -53,9 +74,6 @@ jobs:
         run: |
           composer install --prefer-dist
 
-      - name: Wait for Search server
```

Comment:
You can leave the wait for server script so that cluster is available when the integration tests run.

Reply:
Yea maybe it is the wait for script for the cluster error